Automattic/jetpack
# Jetpack Social: Added a button to the admin page of Jetpack Social to take the user to the Jetpack Cloud screen for managing their connections.

Added a button to the admin page of Jetpack Social to take the user to the Jetpack Cloud screen for managing their connections.

#### Changes proposed in this Pull Request:

 - Call the Publicize package's publicize_connections_url in projects/plugins/social/src/class-jetpack-social.php to get the admin page redirect URL to take the user to Jetpack Cloud. 

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Activate Jetpack Social and the classical editor. Make sure you have some connections added. Make sure Jetpack the plugin deactivated. 
* Create a post and click on the settings in the editor's metabox for Publicize. You should be redirected to Calypso Green. 

<img width="1506" alt="Screenshot 2022-04-29 at 8 09 37 PM" src="https://user-images.githubusercontent.com/6594561/165966961-9381e500-0051-40ce-b10d-0c0e57e26bb6.png">
